src/Components/SimpleForm.js

```diff
@@ -0,0 +1,69 @@
+
+import React, { useState } from "react";
+import {
+  Form,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  ActionGroup,
+  Button,
+  CardBody,
+  Card,
+  Bullseye
+} from "@patternfly/react-core";
+import SimpleEmptyState from "./SimpleEmptyState";
+
+export default function SimpleForm(props) {
+  const [selectProductsVersion, setSelectProductsVersion] = useState("");
+  const [selectLocales, setSelectLocales] = useState("");
+  return (
+    <>
```

Comment:
We can remove React Fragments from here as well.
Also, add a new line at the end of every file.

Reply:
Sure.